README_MACOSX_NOTES.md

```diff
@@ -31,28 +33,12 @@ brew outdated
 brew upgrade
 ```
 
-#### Important:
-If you have mysql < 9.0 installed remove mysql and all vestiges before continuing.
+### Install a bunch of useful stuff from Homebrew
 
-- `ps -ax | grep mysql`
-- stop and kill any MySQL processes
-- `brew remove mysql`
-- `brew cleanup`
-- `sudo rm /usr/local/mysql`
-- `sudo rm /etc/my.cnf`
-- `sudo rm /usr/local/etc/my.cnf`
-- `sudo rm -rf /usr/local/var/mysql`
-- `sudo rm -rf /usr/local/mysql*`
-- `sudo rm ~/Library/LaunchAgents/homebrew.mxcl.mysql.plist`
-- `sudo rm -rf /Library/StartupItems/MySQLCOM`
-- `sudo rm -rf /Library/PreferencePanes/My*`
-- edit /etc/hostconfig and remove the line `MYSQLCOM=-YES-`
-- `rm -rf ~/Library/PreferencePanes/My*`
-- `sudo rm -rf /Library/Receipts/mysql*`
-- `sudo rm -rf /Library/Receipts/MySQL*`
-- `sudo rm -rf /private/var/db/receipts/*mysql*`
+#### WARNING
 
-Install a bunch of useful stuff from `Homebrew`
+**If you have mysql version < 9.0 installed, remove mysql and all its vestiges before
+continuing. See footnote 1.<sup id="a1">[1](#f1)</sup>**
```

Comment:
Joe - I'd add a note that if you have older hardware, you should neither update Homebrew nor MySQL, which will not install in any case on older Macs. Following this step was a 12-hour time-waster for me in Virginia.

Reply:
Thanks for the suggestion. I added a warning near the top of the file.
I also added a Table of Contents to make the file easier to navigate.
I will merge it.